pkg/containerutil/container_network_manager.go

```diff
@@ -260,7 +260,11 @@ func (m *containerNetworkManager) getNetworkingContainerForArgument(ctx context.
 	if err != nil {
 		return nil, err
 	}
-	defer cancel()
+	defer func(){
+		cancel()
+		client.Close()
+	}()
+
```

Comment:
NewNetworkingOptionsManager should be modified to take a client, then no need to create and close another client here